histogram.go

```diff
@@ -57,6 +57,32 @@ type Histogram struct {
 	sum float64
 }
 
+// ConvertToVMRange distributes input value in a range (lower, upper] to a vmRange static buckets
+func ConvertToVMRange(output map[string]float64, value, lower, upper float64) {
```

Comment:
I don't understand how this function is intended to be used. Could you add an example with it's usage?

 As far as I understand, it's possible to create a new histogram, write value into it and use visit method:

```go
var hs metrics.Histogram

hs.Update(value)
hs.VisitNonZeroBuckets()
```

Reply:
Idea of a function is to convert prometheus histogram ranges to VMranges. In an input we have distribution of values in a range without exact values (that's why approach you've suggested doesn't work), so here I just dividing prometheus range proportionally into VM buckets, e.g.:
input range 2..8 contains value 6 and it should be placed into three ranges 0..3, 3..6 and 6..9. According to a logic of a function to a first output range it will put (3-2)/(8-2) * 6 = 1, to a second (6-3)/(8-2) * 6 = 3 and third (8-6)/(8-2) * 6 =  2

Reply:
Seems like this function is here because it relies on many internal consts. The external caller can't possibly know  about these costs, and neither can it assume these costs. 
So the options are the following:
1. Have this function, with a comment it is a helper function
2. Expose internal consts, so the external caller could do the calculations on its own.
3. Copy the consts to the caller codebase with assumption they won't ever change.

What do you think about this?

Reply:
Personally, I'd prefer 3.

Reply:
I'm ok with 3. @AndrewChubatiuk ?